src/main/java/com/example/template/controller/PatientController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.template.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.template.models.Patient;
+import com.example.template.repositories.PatientRepository;
```

Comment:
probably package structure needs to be updated.

Reply:
@adongare Thanks Ashok! Can you please let me know what package structure should it be?